src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,352 @@
+
+  useEffect(() => {
+    dispatch(fetchDappsList())
+    ValoraAnalytics.track(DappExplorerEvents.dapp_screen_open)
+  }, [])
+
+  return (
+    <SafeAreaView style={styles.safeAreaContainer} edges={['top']}>
+      <DrawerTopBar
+        rightElement={
+          <TopBarIconButton
+            testID="DAppsExplorerScreen/HelpIcon"
+            icon={<Help color={colors.onboardingGreen} />}
+            onPress={openSheet}
+          />
+        }
+        scrollPosition={scrollPosition}
+      />
+      {ConfirmOpenDappBottomSheet}
+      <>
+        {!loading && error && (
+          <View style={styles.centerContainer}>
+            <Text style={fontStyles.regular}>{t('dappsScreen.errorMessage')}</Text>
+          </View>
+        )}
+        {categories.length && (
+          <AnimatedSectionList
+            refreshControl={
+              <RefreshControl
+                tintColor={colors.greenBrand}
+                colors={[colors.greenBrand]}
+                style={{ backgroundColor: colors.light }}
+                refreshing={loading}
+                onRefresh={() => dispatch(fetchDappsList())}
+              />
+            }
+            // @ts-ignore TODO: resolve type error
+            ref={sectionListRef}
+            ListFooterComponent={
+              <>
+                {dappsMinimalDisclaimerEnabled ? (
+                  <Text style={styles.disclaimer}>{t('dappsDisclaimerAllDapps')}</Text>
+                ) : null}
+              </>
+            }
+            ListHeaderComponent={
+              <>
+                <DescriptionView
+                  title={t('dappsScreen.title')}
+                  message={t('dappsScreen.message')}
+                />
+                {/* Dapps Filtering*/}
+                <View style={{ paddingTop: Spacing.Thick24 }}>
+                  <ScrollView
+                    horizontal={true}
+                    // Expand the scrollview to the edges of the screen
+                    style={{ marginHorizontal: -Spacing.Thick24 }}
+                    showsHorizontalScrollIndicator={false}
+                    bounces={false}
+                    ref={horizontalScrollView}
+                  >
+                    {/* All Dapps Filter */}
+                    <DappFilterChip
+                      chipFilter={{ id: 'all', name: t('dappsScreen.allDapps') }}
+                      selectedFilter={selectedFilter}
+                      setFilter={setSelectedFilter}
+                      lastChip={false}
+                      key={'all'}
+                    />
+                    {/* Category Filter Chips */}
+                    {categories.map((category, idx) => {
+                      return (
+                        <DappFilterChip
+                          chipFilter={{ id: category.id, name: category.name }}
+                          selectedFilter={selectedFilter}
+                          setFilter={setSelectedFilter}
```

Comment:
nit: can this be `onPress` instead, so that we immediately know how it's used?

Reply:
Fixed in f17dfe22ae5c764d169170c818bfa012e6257f05!